### DISCUSSION TITLE
<slot x={true} /> in +layout.svelte/+page.svelte?

### DISCUSSION THREAD

**Discussion by ioucyf** [sveltejs/kit#8848] [Q&A]
is it possible to pass stuff in layout slots like components? if not? is there something similar?

**Comment by david-plugge** [comment]
Can you give an example on what you want to achieve?

**Reply by ioucyf** [reply]
Nothing particular comes in mind. 

I was just working with a component utilizing this functionality, and became curious if that's also a thing with SvelteKit. I'm aware pages and layouts pass data through load functions (or context/stores), but I just thought to ask if this is a thing, and wanted to be aware of it's potential benefits. :)

**Comment by teemingc** [comment]
You coould create a store or add data to `$page.data` https://kit.svelte.dev/docs/load#$page-data
